pkg/util/test.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"reflect"
+)
+
+func EquateErrors() cmp.Option {
+	return cmp.Comparer(func(a, b error) bool {
```

Comment:
every error wrapping library supports comparing errors with a chain of wrapped errors. Can we avoid this is possible?